mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java

```diff
@@ -245,7 +245,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 // Get the Image from data
 
                 Uri selectedImage = data.getData();
-                String[] filePathColumn = { MediaStore.Images.Media.DATA };
+                String[] filePathColumn = {MediaStore.Images.Media.DATA};
```

Comment:
Here removing space is not needed I think.

Reply:
Same for this too